apps/monday-code-action-template-node/src/controllers/monday-controller.js

```diff
@@ -1,27 +1,30 @@
 const initMondayClient = require('monday-sdk-js');
-const mondayCodeService = require('../services/monday-code-service');
+const { functionTypes } = require('../consts/supported-function-types')
 
-// Accessed through /monday_code/execute_action
-async function executeAction(req, res) {
+const mappedControllerFunctions = Object.freeze({
+  [functionTypes.ACTION]: executeAction,
+  [functionTypes.REMOTE_OPTIONS]: getRemoteListOptions
+})
+
+async function executeAction(req, res, func) {
   const { shortLivedToken } = req.session;
   const { payload } = req.body;
 
   const mondayClient = initMondayClient();
   mondayClient.setToken(shortLivedToken);
 
   try {
-    const response = await mondayCodeService.actionLogic({ requestPayload: payload, monday: mondayClient }) || {};
+    const response = await func({ requestPayload: payload, monday: mondayClient }) || {};
```

Comment:
where's func coming from? I don't see it here

Reply:
it comes from the controller